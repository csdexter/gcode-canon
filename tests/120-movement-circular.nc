(testing rapid movements)
(machine setup)
M05 S600
M09
M23
M49
M69

(turn servos ON)
M17

(control setup)
G15
G17
G23
G40
G49
G50
G69

(in mm)
G21

(absolute coordinates)
G90 F600
G01 X20 Y20 Z20
G02 R5 X30 Y20
G02 R10 X10 Y20

G01 X10 Y10 Z10
G03 R-5 Z5 X15 Y15

