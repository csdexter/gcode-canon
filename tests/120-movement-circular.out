MSG: WAR: Machine servos activated!
MSG: STA: Scanning input for programs (O words)
MSG: WAR: Machine servos activated!
MPOS,20.00,20.00,20.00
MPOS,30.00,20.00,20.00
MPOS,10.00,20.00,20.00
MPOS,10.00,10.00,10.00
MPOS,15.00,15.00,5.00
MPOS,10.00,10.00,5.00
